cpp/src/examples/gcylon/gsorting_plus.cpp

```diff
@@ -0,0 +1,226 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I suggest not having this file. It seems like a test case

Reply:
removed.